frontend/admin/src/.htaccess

```diff
@@ -26,10 +26,6 @@ FallbackResource index.html
     # Send Requests To Front Controller...
     RewriteCond %{REQUEST_FILENAME} !-d
     RewriteCond %{REQUEST_FILENAME} !-f
```

Comment:
Since we removed the RewriteRule these conditions apply to, we might as well delete these as well

Reply:
Thanks, I had no idea what they were for. :grin: